mappings/net/minecraft/server/world/ChunkLevels.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_xuardpja net/minecraft/server/world/ChunkLevels
```

Comment:
I'd love some quick javadoc on the fields (with a little `{@value}` in them).

Reply:
will do that tonight! they're fairly self-explanatory, so I'll keep it short